spec/helper.js

```diff
@@ -217,11 +217,12 @@ global.helper = {
      * @returns {Promise}
      */
     stop: function () {
-        if (this.core) {
-            return this.core.stop();
-        } else {
-            return bluebird.resolve();
-        }
+      var promises = [new Promise(function (resolve) {
+        setTimeout(resolve);
```

Comment:
I was hoping that having a timeout between tests would yield to `nextTick` and `setImmediate` to be called. Incase it is causing race conditions.